README.md

```diff
@@ -1,2 +1,141 @@
 # Hello Teresa
 A collection of `Hello World` ready to deploy on [teresa](https://github.com/luizalabs/teresa-api) PaaS.
+
+## Deploy
+
+Some information to up and run your application on Kubernetes with Teresa.
+
+### Port
+Don't use a fixed port to up your web application, instead,
+read the environment variable `PORT`, for instance:
+
+```go
+port := os.Getenv("PORT")
+if port == "" {
+    port = "5000"
+}
+http.ListenAndServe(fmt.Sprintf(":%s", port), nil)
+```
+
+The deploy process will set this environment variable
+
+### Procfile
+According to [Heroku's docs](https://devcenter.heroku.com/articles/procfile):
+
+```
+A Procfile is a mechanism for declaring what commands are run by your application’s
+dynos on the Heroku platform.
+```
+
+Teresa follows the same principle.
+As an example, a Python application might have the following command on Procfile:
+
+    web: gunicorn -b 0.0.0.0:$PORT -w 3 --pythonpath src myapp.wsgi
+
+
+### Language detection
+When you make a deploy on Teresa you don't need to specify the language of your application,
```

Comment:
`When you deploy an app using Teresa, you don't have to specify your application's language - it'll be automatically detected`